code/extensions/LDAPGroupExtension.php

```diff
@@ -30,6 +41,19 @@ class LDAPGroupExtension extends DataExtension {
 	);
 
 	public function updateCMSFields(FieldList $fields) {
+		$fields->addFieldToTab('Root.Members', new ReadonlyField('GUID'));
+		$fields->addFieldToTab('Root.Members', new ReadonlyField('DN'));
+		$fields->addFieldToTab('Root.Members', new ReadonlyField('IsImportedFromLDAP', 'Is group imported from LDAP/AD?'));
```

Comment:
`_t()`? Probably check if the `LDAPMemberExtension` has this similar string localised as well.